src/main/java/seedu/address/logic/commands/leaderboardcommand/ShowSimpleLeaderboardCommand.java

```diff
@@ -30,6 +30,7 @@ public ShowSimpleLeaderboardCommand(ArrayList<Comparator<Team>> comparators) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        checkNoTeams(model);
         model.setSimpleLeaderboard(comparators);
```

Comment:
I may be wrong on this, how does #setSimpleLeaderboard differ from #setTopK if the k is set to the size of team list? :bowtie:

Reply:
You're right it doesn't differ if k is set to the size of the team list. But they setSimpleLeaderboard is an abstraction and can be used in setTopK method, which is why i dont use setTopK with the size of the team list for the full leaderboard. I'm streamlining this a little more in my next PR.

Reply:
hahaha then maybe you can abstract the method out or (again correct me if im wrong) can I call setTopK with k as the size of team list under my #setSimpleLeaderboard method? :D

Reply:
no no here's the thing: before you can setTopK you need a simple leaderboard to start with. That's why setSimpleLeaderboard is a separate method. This method can then be called within setTopK as well.